app/controllers/doc_controller.rb

```diff
@@ -88,6 +89,10 @@ def ext
 
   private
 
+  def set_caching
+    expires_in 10.minutes, :public => true
+  end
+
```

Comment:
This seems quite sensible. Is there any reason not to apply the caching rules to the entire site, rather than to individual controllers? I'm not familiar enough with Rails to know what's the norm.